modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/pages/hearing_type_and_additional_issues.rb

```diff
@@ -50,10 +50,10 @@ def hearing_type_text
           end
 
           def extra_issues_table_data
-            header = ['A. Specific Issue(s)', 'B. Disagreement Reason', 'C. Date of Decision']
+            header = ['A. Specific Issue(s)', 'B. Disagreement Area', 'C. Date of Decision']
```

Comment:
Sorry I missed this earlier... Isn't this supposed to be "Area of Disagreement"?

Reply:
It can be. It's the additional page table and this column doesn't map to an existing column, so it looks like I kept it short. I can change it. I'll get clarification today. Thanks for looking so closely!!!

Reply:
Nice catch. Yes, I'll change it.